requirements.txt

```diff
@@ -1,5 +1,5 @@
 flask
-flask-sqlalchemy
+https://github.com/pallets/flask-sqlalchemy/archive/master.zip
```

Comment:
It looks like they released 2.4.0 (and subsequently 2.4.1) 8 hours ago, so we can just make sure the version gets upgraded on the deployment.

https://github.com/pallets/flask-sqlalchemy/releases/tag/2.4.0

Reply:
I saw that but 2.4.1 doesn't actually include the fix for count! Which is extremely irritating. But we can at least pin to a commit or something.

Reply:
Yeah, I misread the release, ignore me.